rpcs3/Emu/Cell/Modules/cellSaveData.cpp

```diff
@@ -1303,10 +1303,8 @@ static NEVER_INLINE error_code savedata_op(ppu_thread& ppu, u32 operation, u32 v
 		}
 	}
 
-
-
 	// Create save directory if necessary
-	if (!psf.empty() && save_entry.isNew && !fs::create_dir(dir_path))
+	if (!psf.empty() && save_entry.isNew && !fs::is_dir(dir_path) && !fs::create_dir(dir_path))
```

Comment:
Maybe create_path or check g_tls_error afterwards?

Reply:
Aren't there some corner cases which yield access error when the directory already exists on Windows?

Reply:
I don't remember one, that's weird

Reply:
https://github.com/RPCS3/rpcs3/commit/8f47f9517a35e2419d86872f8662eb23f92f588f

Reply:
Ah, it was when trying to create stuff like C:\ which can throw permission indeed. It shouldn't happen in this context.